index.d.ts

```diff
@@ -0,0 +1,760 @@
+// Definitions by James George (https://github.com/jamesgeorge007)
```

Comment:
I think your git blame is sufficient 😆

Reply:
It is just a convention to credit the author as found within the DefinitelyTyped project.

Reply:
If you would insist I shall remove those credits :+1:

Reply:
@tabrindle ??